core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy

```diff
@@ -40,12 +40,8 @@ l.layout(norefresh:true, permission:app.ADMINISTER, title:my.displayName) {
             }
             st.adjunct(includes: "lib.form.confirm")
         } else {
-            p {
-                _("There are no cloud implementations for dynamically allocated agents installed.")
-                a(href: rootURL + "/pluginManager/available") {
-                    _("Go to plugin manager.")
-                }
-            }
+            p(_("There are no cloud implementations for dynamically allocated agents installed. "))
+            a(href: rootURL + "/pluginManager/available", _("Go to plugin manager."))
```

Comment:
Would be nice if we could apply the "cloud" or "cloud-provider" label filter in URL. Could be a great symbiosis with @daniel-beck 's labeling PR

Reply:
Is there a way to do that currently? all the filtering seems to be done client side from what I can tell

Reply:
Nice idea, but way out of scope.

Reply:
We _could_ link to https://jenkins.io/doc/developer/extensions/jenkins-core/#cloud but that's almost user-hostile. That would be easy enough to do here.

I did something similar in the Build Authorization Monitor due to lack of alternatives.

Reply:
Anyway, can be done in a follow-up